assets/thanos-ruler/thanos-ruler.yaml

```diff
@@ -20,7 +20,9 @@ spec:
     key: alertmanagers.yaml
     name: thanos-ruler-alertmanagers-config
   containers:
-  - name: thanos-ruler
+  - args:
```

Comment:
hmm, container's args generated by the operator would be overwritten, we probably need to add `spec.additionalArgs` to the ThanosRuler CRD first.

Reply:
Created https://github.com/prometheus-operator/prometheus-operator/issues/5260